kubemarine/procedures/check_iaas.py

```diff
@@ -288,6 +288,30 @@ def system_distributive(cluster):
         
         tc.success(results=", ".join(detected_supported_os))
 
+def check_kernel_version(cluster):
+    """
+    This method compares the linux kernel version with the bad version
+    """
+    with TestCase(cluster.context['testsuite'], '015', "Software", "Kernel version") as tc:
+        bad_results = []
+        bad_kernel_ubuntu = ['5.4.0-132-generic']
+        bad_kernel_centos = []
+        group = cluster.nodes['all']
+        result_group = group.sudo('uname -r && cat /etc/os-release', warn=True)
+        for host, results in result_group.items():
+            result = results.stdout.split('\n')
+            if 'Ubuntu' in result[1]:
+                if result[0] in bad_kernel_ubuntu:
+                    bad_results.append(host.original_host)
+            else:
+                if result[0] in bad_kernel_centos:
+                    bad_results.append(host.original_host)
+
+        if len(bad_results) > 0:
+            cluster.log.debug(f"Bad kernel on: {bad_results}")
+            raise TestWarn("Kernel version bad")
```

Comment:
Probably add hint what to do? What kernel is bad and what is good?

Reply:
add more information

Reply:
> What kernel is bad and what is good?

Just `bad kernel` is not informative. I suggest to print what exactly kernel version is bad, and what kernel version is recommended to upgrade to.

Reply:
add valid version kernel in printout